index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles/style.css">
+  <link rel="stylesheet" href="styles/index-style.css">
+
+  <title>Fansite</title>
+
+</head>
+
+<body class="container">
+  <header >
+    <h1>  Guy Fieri Fansite </h1>
+  </header>
+
+  <nav>
+    <ul>
+      <img id="guy-gif"alt="Guy Gif" src="https://boldtv.com/wp-content/uploads/2020/10/guy-fieri-chefs-kiss.gif">   
+      <li>Home</li>
+      <li><a href="pages/facts.html">Facts</a></li>
+      <li><a href="pages/gallery.html">Gallery</a></li>  
+      <p></p>
```

Comment:
Since this element is empty, we should remove it. If the goal is to have some space between the link above and this image, the preference would be to set a margin or padding on one of the elements to create that space.